source/installation/python_install.rst

```diff
@@ -82,7 +87,6 @@ installation was reported.
   pip install --upgrade aplpy
```

Comment:
@astrofrog - did you misread this?  It already says that pip install is not need for astropy, whereas you are commenting on the `aplpy` install line.